idd/Energy+.idd.in

```diff
@@ -2384,12 +2384,23 @@ Site:WaterMainsTemperature,
        \note field is ignored.
        \type real
        \units C
-  N2 ; \field Maximum Difference In Monthly Average Outdoor Air Temperatures
+  N2 , \field Maximum Difference In Monthly Average Outdoor Air Temperatures
        \note If calculation method is CorrelationFromWeatherFile or Schedule, this input
        \note field is ignored.
        \type real
        \units deltaC
        \minimum 0
+  N3 , \field Temperature Multiplier
+       \note If calculation method is Schedule, this input field is ignored.
+       \type real
+       \units dimensionless
+       \default 1
+       \minimum 0
+  N4 ; \field Temperature Offset
+       \note If calculation method is Schedule, this input field is ignored.
+       \type real
+       \units C
```

Comment:
\unites deltaC (for proper SI/IP conversion)

Reply:
Thanks @mjwitte, I would not have thought of that. Fixed.